packages/package-json-lint/src/project-structure/index.js

```diff
@@ -6,7 +6,7 @@ const spawn = require('@npmcli/promise-spawn');
 const IGNORE_FILE_PATH = path.join(process.cwd(), '.packagejsonlintignore');
 
 const getIgnorer = async () => {
-  const ignoreFileContents = await fs.readFile(IGNORE_FILE_PATH, 'utf8');
+  const ignoreFileContents = await fs.pathExists(IGNORE_FILE_PATH) ? await fs.readFile(IGNORE_FILE_PATH, 'utf8') : '';
   return ignore().add(ignoreFileContents);
```

Comment:
If ignoreFileContents is an empty str and added to `ignore`, does it actually do/add anything?